ROADMAP.md

```diff
@@ -0,0 +1,55 @@
+# cellxgene roadmap
+
+We are very exited for _cellxgene_ to become a valuable tool in collaborations
+between computational biologists and experimental biologists working on
+single-cell transcriptomics data. _cellxgene_ is in active developmentis in
+active development, and we've love to include the community as we plan new
+features to work on.
+
+In the future, we plan to improve cellxgene in the ways described below.
+
+Please note: this roadmap may change at any time.
+
+*Last updated: April 11, 2019*
+
+## What we are building now
+
+In the near term, our goal is to enable teams of computational and experimental
+biologists to explore and annotate their single-cell RNA-seq data.
+
+There are 4 key features we plan to implement in the near term.
+
+- Click install and launch
+- Manual annotation workflows
+- Toggle embeddings
+- Gene information
+
+### Click install and launch
+
+The command line interface for installing and launching cellxgene is a barrier
+for users who are not used to Python or using the command line. We plan to
+support installation and launch of cellxgene on Mac and Windows. See
+https://github.com/chanzuckerberg/cellxgene/issues/687 for more details.
```

Comment:
Could we pretty up the issue links a bit? ex. [Issue #697](https://github.com/chanzuckerberg/cellxgene/issues/687)